src/components/Chips/Chips.tsx

```diff
@@ -7,21 +7,26 @@ import { getCSSVar } from "../../services/themes";
 import { elementColorsNames, getElementColor } from "../../utils/colors-vars-map";
 import Avatar from "../Avatar/Avatar";
 import IconButton from "../IconButton/IconButton";
+import Tooltip from "../Tooltip/Tooltip";
 import { getTestId } from "../../tests/test-ids-utils";
 import { ChipsSize } from "./ChipsConstants";
 import { AvatarType } from "../Avatar/AvatarConstants";
 import { SubIcon, VibeComponent, VibeComponentProps } from "../../types";
 import useHover from "../../hooks/useHover/useHover";
 import useSetFocus from "../../hooks/useSetFocus";
-import { ComponentDefaultTestId } from "../../tests/constants";
 import useClickableProps from "../../hooks/useClickableProps/useClickableProps";
+import useIsOverflowing from "../../hooks/useIsOverflowing/useIsOverflowing";
+import { ComponentDefaultTestId } from "../../tests/constants";
 import { BEMClass } from "../../helpers/bem-helper";
 import "../Clickable/Clickable.scss";
 import styles from "./Chips.module.scss";
 
 const CLICKABLE_CSS_BASE_CLASS = "monday-style-clickable";
 const clickableBemHelper = BEMClass(CLICKABLE_CSS_BASE_CLASS);
 
+const showTriggers = [Tooltip.hideShowTriggers.MOUSE_ENTER, Tooltip.hideShowTriggers.FOCUS];
```

Comment:
we need to remember update the tooltip pr after this merged

Reply:
Nothing bad will happen even if we forget to

Reply:
i guess but there will no need in the Tooltip.hideShowTriggers.FOCUS so we can remove it there